src/User.cpp

```diff
@@ -1,11 +1,23 @@
 #include "User.hpp"
 
 User::User(const std::string username, const std::string pswd, const std::string acc_type) {
+    if (acc_type == "employee") permission = 1;
```

Comment:
Probably need the same check here.  I'd suggest just making a separate function that takes an account type and returns it.  Or, you could create an enum class somewhere and require that as a param for the account type rather than a string, and then you could probably skip the validation.

I.e, somewhere like ```User.hpp```:

```
enum class Account { EMPLOYEE, MANAGER, OWNER };
```

But, since you've already implemented it with a string, I'm fine with that -- I would just like some validation that doesn't let us set fake account types.

So, you could just take that code snippet I gave above, put it in a function somewhere, and use that everywhere we need, i.e:

```
int getAccountPermissionLevel(std::string account) {
    if (account == "employee") {
        p = 1;
    } else if (account == "manager") {
        p = 3;
    } else if (account == "owner") {
        p = 5;
    } else {
        Logger::logError("No such account type '%s'.", account.c_str());
        p = -1;
    }
    return p;
}
```